jib-core/src/main/java/com/google/cloud/tools/jib/JibSystemProperties.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
```

Comment:
No _All rights reserved_

Reply:
Oh, looks like there are other places with "All rights reserved" too. I'll fix this one and fix the rest in a followup.